### ISSUE TITLE
Issue: v0.13.0 of redwood generate auth CLI fails to add the getCurrentUser import

### ISSUE THREAD

**Issue by ericxjohnston** [redwoodjs/redwood#830]
Issue: Running `yarn rw g auth [provider]` in an app created by v0.13.0 of `yarn create redwood-app [directory]` fails to add `import { getCurrentUser } from 'src/lib/auth.js'` as part of the generator command to /api/src/functions/graphql.js

Previously, this worked in version v0.12.0.

The issue is happening because the Redwood CLI generate auth command looks for a `/^(.*importAll.*)$/m,` regex match to add the `getCurrentUser` import (https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/generate/auth/auth.js#L96-L100) but the `importAll` macro was replaced by a babel plugin import in the v0.13.0 release (https://github.com/redwoodjs/create-redwood-app/commit/bc4e93d67a6ef0e16bc3658ae26c3dc78678c702).

And this then causes the currentUser object to return as an array and fail to get logged in user email when the useAuth() hook is used.
```
import { useAuth } from '@redwoodjs/auth'
const { logIn, logOut, isAuthenticated, currentUser } = useAuth()
...
{ isAuthenticated && <li>{currentUser.email}</li> }
```
> The above useAuth() code is from the tutorial authentication setup instructions:
> https://redwoodjs.com/tutorial/authentication#web-authentication:~:text=One%20more%20touch%3A%20let's%20show%20the,about%20the%20currently%20logged%20in%20user%3A

Note, I wasn't sure if this issue belongs in the `create-redwood-app` or `redwood` repos so I added the issue in both. Please use one and close the other. Refer to https://github.com/redwoodjs/create-redwood-app/issues/77

**Comment by jtoar** [comment]
@ericxjohnston Thanks for diagnosing this one :+1:, it made it really easy to provide the fix (I just opened #832 to close it). And this repo is the right one as the generator code is in the cli package.

**Comment by ericxjohnston** [comment]
Awesome. 🎉 Thanks @jtoar .
